corehq/motech/openmrs/templates/openmrs/importers.html

```diff
@@ -1,30 +1,9 @@
-{% extends "domain/admin/info_base.html" %}
+{% extends "hqwebapp/base_section.html" %}
 {% load hq_shared_tags %}
 {% load crispy_forms_tags %}
 {% load i18n %}
 
-{% block stylesheets %}{{ block.super }}
-    <link rel="stylesheet" href="{% static 'codemirror/lib/codemirror.css' %}" />
-    <link rel="stylesheet" href="{% static 'codemirror/addon/fold/foldgutter.css' %}"/>
-    <style>
-        .CodeMirror {
-            border: 1px solid #ccc;
-            height: auto;
-            max-width: 800px;
-        }
-    </style>
-{% endblock %}
-
-{% block js %}{{ block.super }}
-    <script src="{% static 'codemirror/lib/codemirror.js' %}"></script>
-    <script src="{% static 'codemirror/mode/javascript/javascript.js' %}"></script>
-    <script src="{% static 'codemirror/addon/fold/foldcode.js' %}"></script>
-    <script src="{% static 'codemirror/addon/fold/foldgutter.js' %}"></script>
-    <script src="{% static 'codemirror/addon/fold/brace-fold.js' %}"></script>
-
-    <script src="{% static 'userreports/js/base.js' %}"></script>
```

Comment:
@proteusvacuum @kaapstorm This inclusion implies that this page should have some code editor windows, and there are indeed a couple of json fields...but on prod, those fields just appear as plain text areas. Could one of you verify whether or not prod is working as expected?

Reply:
I don't think those are working correctly. I think a similar change to this one needs to be made: https://github.com/dimagi/commcare-hq/pull/21093

Reply:
will let @kaapstorm confirm though, as he has more context on those pages than me.

Reply:
figured it out: https://github.com/dimagi/commcare-hq/pull/21277